docker/cardano/README.md

```diff
@@ -0,0 +1,23 @@
+# Cardano docker scripts
+
+## Basic Testnet
+
+This creates a testnet with three nodes and a user account with all the funds.
+It also runs a custom version of cardano-wallet server, accessible via port 8190,
+which includes the features needed by the force-bridge collector and
+verifier operators.
+
+```
+docker-compose build
```

Comment:
build this from scratch is a big overhead.
It took hours for me and failed.
Could you plz separate the build part and usage part, and push the built images to dockerhub for easier testing?

Reply:
@huwenchao I have modified the code to make use of prebuilt docker images.